flutter-studio/src/io/flutter/profiler/FlutterAllMemoryData.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (C) 2018 The Android Open Source Project
```

Comment:
Seems like the wrong copyright notice

Reply:
Switched to Chromium copyright instead of Android Studio copyright.

Reply:
Changed to Chromium copyright.